src/main/java/xades4j/verification/TimeStampVerifierBase.java

```diff
@@ -76,23 +75,11 @@ public final QualifyingProperty verify(
             setTimeMethod.invoke(prop, ts);
             return prop;
         }
-        catch(UnsupportedAlgorithmException ex)
-        {
-            throw getEx(ex, this.propName);
-        }
-        catch (CannotAddDataToDigestInputException ex)

```

Comment:
Keep this one, because the thrown exception is not of the same type.

Reply:
Oh! I oversaw this one! Thanks, good eyes!